package.json

```diff
@@ -29,5 +29,9 @@
       "packages/**/test.js",
       "packages/**/test/*.js"
     ]
+  },
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "test": "ava"
```

Comment:
Most of my machines do not have "make" installed.
(I didn't even notice the Makefile until you mentioned it.)

Also, I have editor plugin that detects "test" script existence in package.json, and runs tests in background after every save, reporting success/failure in statusbar.